src/epanet2.h

```diff
@@ -0,0 +1,229 @@
+/*
```

Comment:
Let's exclude any epanet headers and build libraries from this repo, as those are available elsewhere.